app/assets/stylesheets/forever_style_guide/modules/_nav.scss

```diff
@@ -588,29 +580,247 @@ a.navigation-user_info-logged_out-log_in_link {
 }
 
 // Overriding button styles to make it look good with smaller font size
-.navigation-user_info-logged_out-sign_up_btn {
-  margin-left: $padding-small-horizontal;
-  height: auto !important;
+a.navigation-user_info-logged_out-sign_up_btn {
+  @media screen and (max-width: $screen-mobile-menu) {
+    width: 120px;
+  }
+  
+  display: -webkit-box;
+  margin-left: 4px;
+  margin-right: $padding-10;
 
   @media screen and (min-width: $screen-md) {
     font-size: $font-size-100 !important;
-    letter-spacing: $letter-spacing-large;
+    margin-left: 3px;
   }
 
   @media screen and (min-width: $screen-lg) {
-    margin-left: $padding-large-horizontal;
     font-size: $font-size-200 !important;
+    margin-left: 3px;
   }
 }
 
 .user_info-logged_in-account_dropdown-menu-list-sign_out {
   color: $color-danger !important;
 }
 
-.letter-spacing-small {
-  letter-spacing: 0.3px;
+ @media screen and (max-width: 768px) {
+  .collapsing
+  {
+      position: absolute !important;
+      z-index: 20;
+      width: 100%;
+      top: 50px;
+      background: white;
+  }
+  .collapse.in {
+      display: block;
+      position: absolute;
+      z-index: 20;
+      width: 100%;
+      top: 50px;
+      background: white;
+  }
+  .navbar-collapse
+  {
+    max-height: none !important;
+  }
+}
+
+.navigation-menu-list_item:not(:last-of-type) {
+  border-bottom: none;
+}
+
+.navbar-nav > li.navigation-user_info-logged_out.navigation-log_in {
+  padding-top: 0;
+}
+
+.navigation-menu-list .navbar-nav > li > a.navigation-user_info-logged_out-sign_up_btn:hover {
+    color: $color-primary;
+}
+
+.nav-sign_in-sign_up {
+  @media (max-width: $screen-xs) {
+    float: left !important;
+  }
 }
 
-.letter-spacing-large {
-  letter-spacing: 0.5px;
+.nav-mobile-menu {
+  @media (max-width: $screen-mobile-menu) {
+    width: 0;
+    height: 100%;
+    background-color: $color-gray-100; /* Black fallback color */
+    overflow: hidden;
+    position: fixed; /* Stay in place */
+    transition: 0.4s;
+    right: 0;
+    border-left: solid 1px $color-gray-200;
+    opacity: 0.5;
+    max-width: 320px;
+  }
+
+  @media (max-width: $screen-xs) {
+    max-width: 100%;
+  }
+}
+
+.nav-mobile-menu.top-offset {
+  @media (max-width: $screen-mobile-menu) {
+    top: 80px;
+  }
+}
+
+.nav-mobile-menu-account.visible-mobile {
+  @media (max-width: $screen-xs) {
+    top: 119px;
+  }
+}
+
+.nav-mobile-menu.in {  
+  height: 100%;
+  width: 100%;
+  z-index: $zindex-tooltip; /* Sit on top */
+  right: 0;
+  overflow-x: hidden; /* Disable horizontal scroll */
+  overflow-y: auto;
+  transition: 0.5s;
+  opacity: 1;
+  border-left: solid 1px $color-gray-200;
+  box-shadow: $navigation-shadow;
+
+  @media (max-width: $screen-mobile-menu) {
+    padding-top: 15px;
```

Comment:
This padding should be removed b/c I think it's causing a weird transition effect.